### ISSUE TITLE
SSH connection terminated right after logon

### ISSUE THREAD

**Issue by mareq** [mareq/docker_rotating-machines#1]
The SSH connection is opened, user is asked for password, after entering it, /etc/motd is printed and connection gets closed with message `Connection to localhost closed.`. SSH log says the following:

```
Server listening on 0.0.0.0 port 22.
Server listening on :: port 22.
Accepted password for [User] from [IP Address] port [Port] ssh2
PAM: pam_open_session(): Cannot make/remove an entry for the specified session
Received disconnect from [IP Address]: disconnected by user
```

**Comment by mareq** [comment]
Fix for the problem can be found here (`Dockerfile` needs to be adjusted):
- https://github.com/docker/docker/issues/5663
- https://github.com/tweag/lustre-boxes/commit/1d8ad62bf5fa984673ed94dec2b2cab73f32667c

**Comment by mareq** [comment]
Fixed by cc393af57eb374cf169ee5cfbcac918e32364432.
